test/sql-tap/gh-6024-funcs-return-bin.c

```diff
@@ -29,7 +35,7 @@ ret_uuid(box_function_ctx_t *ctx, const char *args, const char *args_end)
 	memset(&uuid, 0x11, sizeof(uuid));
 	char res[BUF_SIZE];
 	memset(res, '\0', BUF_SIZE);
-	char *end = mp_encode_uuid(res, &uuid);
+	char *end = tnt_mp_encode_uuid(res, &uuid);
```

Comment:
If you'll not change it, everything will work, right?

Reply:
No, it won't work because I am using `tnt_mp_encode_decimal` and `tnt_mp_encode_uuid` which are exported from `tarantool`. I do this because otherwise I need to link with `core` (which must be rebuilt with` -fPIC`), because `core` is used in` tt_uuid.c`, which is used in `mp_uuid.c`.

Reply:
Got it: it is not just msgpuck symbol, but rather tarantool symbol. And the symbol will be resolved at loading this code as a dynamic library by tarantool. Okay so.